tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockListEditor/blockListEditorDocument.spec.ts

```diff
@@ -0,0 +1,176 @@
+﻿import {AliasHelper, ConstantHelper, test} from "@umbraco/playwright-testhelpers";
+import {DocumentTypeBuilder} from "@umbraco/json-models-builders";
+import {expect} from "@playwright/test";
+import {BlockListDataTypeBuilder} from "@umbraco/json-models-builders/dist/lib/builders/dataTypes";
+
+test.describe('BlockListEditorDocument', () => {
+  const documentName = 'DocumentName'
+  const elementName = 'TestElement';
+  const blockListName = 'BlockListTest';
+
+  test.beforeEach(async ({page, umbracoApi, umbracoUi}) => {
+    await umbracoApi.login();
+  });
+
+  async function createElement(umbracoApi, elementName) {
+    const elementAlias = AliasHelper.toAlias(elementName);
+    const elementType = new DocumentTypeBuilder()
+      .withName(elementName)
+      .withAlias(elementAlias)
+      .AsElementType()
+      .addGroup()
+        .withName("TestString")
+        .withAlias('testString')
+        .addTextBoxProperty()
+          .withLabel("Title")
+          .withAlias("title")
+        .done()
+      .done()
+      .build();
+    await umbracoApi.documentTypes.save(elementType);
+
+    return elementType;
+  }
+
+  test('can create empty block list in a document', async ({page, umbracoApi, umbracoUi}) => {
+    const groupName = 'blockListGroup';
+
+    await umbracoApi.documentTypes.ensureNameNotExists(documentName);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockListName);
```

Comment:
I think these are used in all tests, if they are, we should call them in `beforeEach` & `afterEach` hooks instead